monitoring/uss_qualifier/suites/faa/uft/design/README.md

```diff
@@ -0,0 +1,79 @@
+# UFT Message Signing tests for SCD using Uss_Qualifier
+
+UTM Field Test (UFT) has a requirement for USSes to sign their message requests
+and responses in the SCD flow using IATF certificates provided by FAA.
+
+This test suite helps to test the message signing by extending [suites.astm.utm.f3548_21](../../../astm/utm/f3548_21.yaml)
+with scenarios that trigger reporting of message signing in the interactions with mock_uss.
+
+## Test Harness Setup
+
+The test setup includes running the following components on your local machine, as also shown in the [diagram](./InterUss%20Test%20Harness%20With%20Message%20Signing.png)
+
+1. Uss Qualifier - This is the test driver that injects the operations for the SCD flow tests in USSes.
+2. Dummy Auth - This is a dummy auth server that provides tokens for uss-to-uss and uss-to-dss communication. Runs as a dockerized container at port 8085.
+3. DSS - This is a local DSS running in dockerized container, and is available at port 8082.
+4. Mock USS - This is a mock implementation of a USS running as a dockerized container at port 8074. The message signing reports are generated using mock_uss.
+5. USS-under-Test - This is the USS that needs to be tested.
+6. Uss Qualifier Interface. USSes need to develop an interface for their USS
+to interface with the test harness. Uss_Qualifer will inject operations through this interface. The spec to
+implement is - [Strategic Coordination Test Data Injection](https://github.com/interuss/automated_testing_interfaces/blob/fa3a5f544161c408f50255630a23b670c74a67d1/scd/v1/scd.yaml)
+
+The main idea behind the tests is that mock_uss will consume and validate all the requests and responses from USS-under-test.
+
+Note - As different USSes have different implementations, it could happen that you might need to create a subscription in the area of the flights.
+
+## Steps to run the test
+
+1. Set your UssQualifier Interface implementation url in the [configuration file ](../../../../configurations/dev/faa/uft/local_message_signing.yaml )to run the UFT message signing tests.
```

Comment:
I'd recommend having the user first copy configurations/dev/faa/uft/local_message_signing.yaml to, e.g., configurations/personal/message_signing.yaml before making personalized changes